package.json

```diff
@@ -44,6 +44,7 @@
     "eslint-plugin-prettier": "^3.0.0",
     "eslint-plugin-react": "^7.19.0",
     "fs-extra": "^10.0.0",
+    "grommet": "^2.19.1",
```

Comment:
We've tried to avoid circular dependencies, since `grommet` depends on `grommet-theme-hpe` already.